tests/aruba/aruba.md

```diff
@@ -0,0 +1,50 @@
+# Aruba-rspec testframework for CB CLI
+The aruba test project is located in tests/aruba/
+
+## E2E tests
+
+Run without Docker:
+```
+curl -sSL https://get.rvm.io | bash -s stable --ruby
+source /Users/mhalmy/.rvm/scripts/rvm
+gem install rspec
+gem install aruba
+gem install aruba-rspec
+gem install json
+gem install rspec-json_expectations
+gem install rspec_junit_formatter
+#Edit localvars
+source localvars
+
+rspec spec/e2e/cluster.rb  # Run a specific test
+rspec spec/e2e/*.rb # Run all tests
+rspec -f RspecJunitFormatter -o test-result.xml -f h spec/e2e/*.rb | tee test-result.html | ruby -n # Run with output files
+```
+Run with Docker locally:
+```
+#Edit localvars
+source localvars
+#run script: 
+scripts/docker-e2e-test-aruba-local.sh
+Check result in:
+	out.html
+```
+## Integration test
+
+Run all test cases:
+```
+rspec -f RspecJunitFormatter -o test-result.xml -f h spec/integration/*.rb | tee test-result.html | ruby -n spec/common/integration_formatter.rb
+```
+## Hints for test cases
+
+Test cases use a commandbuilder to run the CB CLI commands.
+Examples:
+
+```
+cb.credential.describe.name(“test”).build
+```
+Or use the builds method to hide sensitive data in the generated report file.
+```
+cb.credential.create.openstack.keystone_v2.name(@os_credential_name).tenant_user(ENV['OS_V2_USERNAME']).
+tenant_password(ENV['OS_V2_PASSWORD']).tenant_name(ENV['OS_V2_TENANT_NAME']).endpoint(ENV['OS_V2_ENDPOINT']).builds
+```
```

Comment:
When should I apply these commands?

Before test run or should I place these in a test file somewhere? These are mandatory for test execution? The purpose and goal of these is not clear for me.